app/models/verification/bed_verification.rb

```diff
@@ -10,7 +10,14 @@ def self.included(base)
 
   def validate_and_create_audits?(params)
     return false unless valid?
+
     params[:source_plates] = parse_source_and_destination_barcodes(scanned_values).flatten.join(' ')
-    return super(params)
+    super(params)
+  end
+
+  # Returns the information for all scans. Empty beds are filtered out.
+  def metadata(params)
+    scanned_beds = params[:robot].reject { |_k, v| v.values.all?(&:blank?) }
+    { 'scanned' => scanned_beds }
   end
```

Comment:
Spits out the scanned information, filtering out any beds which were left completely empty.